src/hii_popdens.py

```diff
@@ -11,8 +11,19 @@ class HIIPopulationDensity(EETask):
             "ee_type": EETask.IMAGECOLLECTION,
             "ee_path": "CIESIN/GPWv411/GPW_Population_Density",
             "maxage": 5  # years
-        }
-    }
+        },
+        "ocean": {
+            "ee_type": EETask.IMAGE,
+            "ee_path": "users/aduncan/cci/ESACCI-LC-L4-WB-Ocean-Map-150m-P13Y-2000-v40",
+        },
+        "jrc": {
+            "ee_type": EETask.IMAGE,
+            "ee_path": 'JRC/GSW1_0/GlobalSurfaceWater',
+        },
+        "caspian": {
+            "ee_type": EETask.FEATURECOLLECTION,
+            "ee_path": 'users/aduncan/caspian',
```

Comment:
See above -- though I think these could be just combined into one mask that is used everywhere no?